go/classes/classes.go

```diff
@@ -50,6 +50,7 @@ const (
 // Form and input and related classes
 const (
 	Form          = "popui-form"
+	FormNoSpacing = "popui-form popui-form__nospacing"
```

Comment:
Just looking at the class feels wrong and indicates to me that forms shouldn't be used for styling. My suggestion would be to assume that forms have no styling by default, and build another element that does. Perhaps you want a `Fieldset` for example with different styling, as that implies something visual and could be re-used for blocks (caution would need to be taken to allow embedding of Fieldsets within Fieldsets.)